base/exports.jl

```diff
@@ -62,7 +62,7 @@ export
     Enumerate,
     Factorization,
     FileMonitor,
-    FloatRange,
+    StepRangeLen,
```

Comment:
I guess FloatRange didn't have a docstring? if you want the new one to go into the manual, it should be added to one of the stdlib doc index files